random-fu/src/Data/Random/Lift.hs

```diff
@@ -5,7 +5,6 @@ module Data.Random.Lift where
 import Data.RVar
```

Comment:
https://github.com/haskell-numerics/random-fu/issues/47 - sadly the OP never explained the reason for their beef

Reply:
From my understanding `Lift` is possible because `PromtT` doesn't rely on the underlying monad and this `lift` allows moving between them. Honestly though, I have no opinion on the matter since I never used it and don't really know use cases where it can be useful.

Reply:
https://github.com/haskell-numerics/random-fu/issues/73